### ISSUE TITLE
Computers locked

### ISSUE THREAD

**Issue by toddgallagher** [veyon/veyon#422]
The user computers randomly lock. Nothing can be done from them but I can still control them from mine. I do not have them on lock either. They can still see the screens just no control.

This has happened several times.

**Comment by toddgallagher** [comment]
32 bit. These are laptops. Even with a mouse it is disabled. I remedied it by going through safe boot and installing 4.0.5.0 on cmd which someone else had recommended on this board. 

The drivers were disabled for the keyboard and trackpad when I checked through the master. So something is causing them to be under lock when they are not but the screen is still showing.

**Comment by toddgallagher** [comment]
The operating system is Windows 7 Service Pack 1.

**Comment by tobydox** [comment]
Does the issue still happen with 4.1.4?

**Comment by toddgallagher** [comment]
I have not tried 4.1.4. I have 4.1.2. I will make sure I update them and see.

**Comment by SARAH12345677** [comment]
TRY TO RESTART THE PROGRAM AND TRY IT AGAIN

*DAVE BUSH*

**Comment by dxdemetriou** [comment]
It's happening on 4.1.4 too.
Related (same) posts:
https://github.com/veyon/veyon/issues/154
https://github.com/veyon/veyon/issues/439

**Comment by SARAH12345677** [comment]
WHAT DO YOU MEAN ITS LOCK?
-DAVE BUSH-

JOHN 3:16
For God so loved the world, that he gave his only Son, that whoever
believes in him should not perish but have eternal life.

**Comment by tobydox** [comment]
Is this still an issue with 4.2.x?

**Comment by dxdemetriou** [comment]
I've just started trying Veyon 4.2.2 (after of migration from 4.1.6 to 4.2.2 and re-enabling of the interception driver). I tried it with PS2 & USB Keyboard/Mouse on Windows 7 64bit without any problems yet. I'll make more checks on Windows 7 32bit / Windows 10 and on more computers during the summer period, and I'll post here if everything is ok or not. One reason of the problem were the crashing of Master on Windows 7 32bit (I haven't tried it yet). Thanks.
